### DISCUSSION TITLE
Image Resize Failed

### DISCUSSION THREAD

**Discussion by programmerarunkumar** [ImageMagick/ImageMagick#2445] [Help]
I tried to resize the image with Magick Wand method

MagickWandGenesis();
IsMagickWandInstantiated();
NewMagickWand();
MagickReadImage();
MagickResizeImage();
MagickWriteImage();
DestroyMagickWand();
MagickWandTerminus();

Used this methods in C language to resize the image 
Called this C Code  from Java 

Resize Failed for some Images , why ?

**Comment by snibgo** [comment]
Do you trap errors and show error messages? What are the error messages?

**Reply by programmerarunkumar** [reply]
resized failed for some of the images & those original images are corrupted 
https://drive.google.com/file/d/1Q3FtPpje6eVJbAP53bLvtvI4VNUZAkhE/view?usp=sharing
my source code where resized failed
